### ISSUE TITLE
[Bug]: @carbon/charts-angular library do not work with latest node.js (16.17.0)

### ISSUE THREAD

**Issue by plachy-jozef** [carbon-design-system/carbon-charts#1445]
### Contact Details

jozef.plachy@sk.ibm.com

### What happened?

`npm i` is resulting to peerDependancy error instead of installation.

`npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: carbon-charts-angular-13@0.0.2
npm ERR! Found: rxjs@7.5.6
npm ERR! node_modules/rxjs
npm ERR!   rxjs@"~7.5.5" from the root project
npm ERR!   peer rxjs@"^6.5.3 || ^7.4.0" from @angular/common@13.3.11
npm ERR!   node_modules/@angular/common
npm ERR!     @angular/common@"~13.3.2" from the root project
npm ERR!     peer @angular/common@"^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0" from @carbon/charts-angular@1.0.1
npm ERR!     node_modules/@carbon/charts-angular
npm ERR!       @carbon/charts-angular@"1.0.1" from the root project
npm ERR!   1 more (@angular/core)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer rxjs@"^6.0.0" from @carbon/charts-angular@1.0.1
npm ERR! node_modules/@carbon/charts-angular
npm ERR!   @carbon/charts-angular@"1.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\A69749693\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\A69749693\AppData\Local\npm-cache\_logs\2022-09-16T12_57_14_617Z-debug-0.log`

When forcing installation with on of error suggested flags the **css of built app is not working properly** resulting of donut buttons malfunction.

STEPS to reproduce:
I have cloned this example project to local:
https://github.com/nstuyvesant/carbon-charts-angular-13

I have used Node version 16.14.1 as advice in npm message during install.
I have updated carbon-chart-angular version to 1.0.1 in package.json (this is the first one that starts to behave with the issue, all prior versions working without this issue, all later versions have this issue).

I have init new repo and copied there used code. Here is the url:
https://github.com/plachy-jozef/carbon-chart-angular-issue

### Version

Node v. 16.14.1.
@carbon/charts-angular@1.0.1
Angular 13.3.11 (latest)
Angular-cli 13.3.9 (latest)

### Data & options used

```js
https://github.com/nstuyvesant/carbon-charts-angular-13
```

### Relevant log output

```js
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: carbon-charts-angular-13@0.0.2
npm ERR! Found: rxjs@7.5.6
npm ERR! node_modules/rxjs
npm ERR!   rxjs@"~7.5.5" from the root project
npm ERR!   peer rxjs@"^6.5.3 || ^7.4.0" from @angular/common@13.3.11
npm ERR!   node_modules/@angular/common
npm ERR!     @angular/common@"~13.3.2" from the root project
npm ERR!     peer @angular/common@"^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0" from @carbon/charts-angular@1.0.1
npm ERR!     node_modules/@carbon/charts-angular
npm ERR!       @carbon/charts-angular@"1.0.1" from the root project
npm ERR!   1 more (@angular/core)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer rxjs@"^6.0.0" from @carbon/charts-angular@1.0.1
npm ERR! node_modules/@carbon/charts-angular
npm ERR!   @carbon/charts-angular@"1.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\A69749693\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\A69749693\AppData\Local\npm-cache\_logs\2022-09-16T12_57_14_617Z-debug-0.log
```

### Codesandbox example

https://github.com/plachy-jozef/carbon-chart-angular-issue

**Comment by plachy-jozef** [comment]
I have used node v. 16.17.0 in the name of the issue because our current project running in docker started to use this version. During my investigation I found out the advice of the highest compatible node that should work and that is 16.14.1. 
I started with carbon-chart-version library with v. 0.56.0 as used in the example from which I took code and started to incrementally updated it.
The version 1.0.1 was the first one which began start to have peedDependancy error and forcing the install would end in app with malfunctioned css.
